python/ray/tune/logger/json.py

```diff
@@ -41,8 +41,8 @@ class JsonLogger(Logger):
 
     def _init(self):
         self.update_config(self.config)
-        local_file = os.path.join(self.logdir, EXPR_RESULT_FILE)
-        self.local_out = open(local_file, "a")
+        local_file = Path(self.logdir, EXPR_RESULT_FILE)
```

Comment:
Should this include `as_posix()`?

Reply:
Same question for other instances below.

Reply:
I just used `Path.open` instead of `open(str)` below.